Source/Rows/SwitchRow.swift

```diff
@@ -83,7 +83,7 @@ open class SwitchRow<T: SwitchCell>: SwitchRowCompatible, Equatable {
   public let cellReuseIdentifier: String = T.reuseIdentifier
 
   /// The cell style is `.default`.
-  public let cellStyle: UITableViewCell.CellStyle = .default
+  public let cellStyle: UITableViewCell.CellStyle = .subtitle
```

Comment:
Thanks for spotting this bug!

To make it also compatible with `.value1` and `.value2`, perhaps it could be:

```suggestion
  public var cellStyle: UITableViewCell.CellStyle {
    return detailText?.style ?? .subtitle
  }
```